### ISSUE TITLE
docs: list of providers that work or don't work out of the box

### ISSUE THREAD

**Issue by venikx** [notthebee/ansible-easy-vpn#107]
Hi,

Foremost, thank you for your work here. Really appreciate how easy it was to set up the VPN (turns out my problems were provider related), and for someone who currently reading more about Ansible a nice resource to learn from. 

Story:

> I initially went with a certain provider (not the RackNerd one) as I wanted the server to be in Europe. It turns out that on that provider's Ubuntu image the Ansible playbook run into some issues (I had to update and upgrade, before running the playbook) and had weird Python errors. Aside from this, I also had problems with actually getting internet through the VPN, which ultimately was resolved by restarting the wg-easy docker container. 

> Eventually, I ran into issues again (same problem came back a day later) so, I went ahead and bought a VPS (the same RackNerd one from the video, but in Europe which costed a bit more), and installed it first try and was able to connect first try. 

Would you be looking into providing a list with recommended and non-recommended providers from https://lowendstock.com/? Or is that outside the scope of this repository?

**Comment by notthebee** [comment]
Hey,

It would be very helpful if you told me what VPS provider you went with. If I have time, I might get a VPS from them and test my script on it.

Thanks!

**Comment by venikx** [comment]
> Hey,
> 
> It would be very helpful if you told me what VPS provider you went with. If I have time, I might get a VPS from them and test my script on it.
> 
> Thanks!

alexhost.com

**Comment by DoctorHoax** [comment]
> I also had problems with actually getting internet through the VPN, which ultimately was resolved by restarting the wg-easy docker container.
> 
Hi, how did you restart wg-easy docker container? 
I have the same issue, wireguard connects but I can't access the internet through it.
Although I live in Iran and guess that the government might have filtered some default ports, I'd appreciate it if you tell me how to customize the ports too.

**Comment by notthebee** [comment]
> > I also had problems with actually getting internet through the VPN, which ultimately was resolved by restarting the wg-easy docker container.
> 
> Hi, how did you restart wg-easy docker container? I have the same issue, wireguard connects but I can't access the internet through it. Although I live in Iran and guess that the government might have filtered some default ports, I'd appreciate it if you tell me how to customize the ports too.

>>> #111

**Comment by notthebee** [comment]
> > Hey,
> > It would be very helpful if you told me what VPS provider you went with. If I have time, I might get a VPS from them and test my script on it.
> > Thanks!
> 
> alexhost.com

Finally got to run the script on an AlexHost VPN.
They seem to automatically run `apt-get dist-upgrade` after the VPS is provisioned. Pretty weird.
When I tried to run the script I got an error about a dpkg lock, which went away after I waited for the `dist-upgrade` to finish.

Thank you for letting me know. I'll start documenting the VPS providers in README.md
